
From: Martin Schwidefsky <schwidefsky@de.ibm.com>

The CLOCK_TICK_FACTOR and FINETUNE defines from <asm/timex.h> are not used
anywhere.  Kill them.

Signed-off-by: Martin Schwidefsky <schwidefsky@de.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-arm/arch-lh7a40x/timex.h |    1 -
 25-akpm/include/asm-arm/arch-sa1100/timex.h  |    1 -
 25-akpm/include/asm-h8300/timex.h            |    4 ----
 25-akpm/include/asm-i386/timex.h             |    4 ----
 25-akpm/include/asm-m68k/timex.h             |    4 ----
 25-akpm/include/asm-ppc/timex.h              |    4 ----
 25-akpm/include/asm-ppc64/timex.h            |    4 ----
 25-akpm/include/asm-s390/timex.h             |    4 ----
 25-akpm/include/asm-sh/timex.h               |    4 ----
 25-akpm/include/asm-sparc/timex.h            |    4 ----
 25-akpm/include/asm-sparc64/timex.h          |    4 ----
 25-akpm/include/asm-v850/timex.h             |    4 ----
 25-akpm/include/asm-x86_64/timex.h           |    4 ----
 13 files changed, 46 deletions(-)

diff -puN include/asm-arm/arch-lh7a40x/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-arm/arch-lh7a40x/timex.h
--- 25/include/asm-arm/arch-lh7a40x/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.282380296 -0700
+++ 25-akpm/include/asm-arm/arch-lh7a40x/timex.h	2004-09-20 11:30:28.303377104 -0700
@@ -14,5 +14,4 @@
 
 /*
 #define CLOCK_TICK_RATE		3686400
-#define CLOCK_TICK_FACTOR	80
 */
diff -puN include/asm-arm/arch-sa1100/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-arm/arch-sa1100/timex.h
--- 25/include/asm-arm/arch-sa1100/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.284379992 -0700
+++ 25-akpm/include/asm-arm/arch-sa1100/timex.h	2004-09-20 11:30:28.303377104 -0700
@@ -10,4 +10,3 @@
  * SA1100 timer
  */
 #define CLOCK_TICK_RATE		3686400
-#define CLOCK_TICK_FACTOR	80
diff -puN include/asm-h8300/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-h8300/timex.h
--- 25/include/asm-h8300/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.286379688 -0700
+++ 25-akpm/include/asm-h8300/timex.h	2004-09-20 11:30:28.303377104 -0700
@@ -7,10 +7,6 @@
 #define _ASM_H8300_TIMEX_H
 
 #define CLOCK_TICK_RATE CONFIG_CPU_CLOCK*1000/8192 /* Timer input freq. */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long cycles_t;
 extern short h8300_timer_count;
diff -puN include/asm-i386/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-i386/timex.h
--- 25/include/asm-i386/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.287379536 -0700
+++ 25-akpm/include/asm-i386/timex.h	2004-09-20 11:30:28.303377104 -0700
@@ -15,10 +15,6 @@
 #  define CLOCK_TICK_RATE 1193182 /* Underlying HZ */
 #endif
 
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 /*
  * Standard way to access the cycle counter on i586+ CPUs.
diff -puN include/asm-m68k/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-m68k/timex.h
--- 25/include/asm-m68k/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.289379232 -0700
+++ 25-akpm/include/asm-m68k/timex.h	2004-09-20 11:30:28.304376952 -0700
@@ -7,10 +7,6 @@
 #define _ASMm68k_TIMEX_H
 
 #define CLOCK_TICK_RATE	1193180 /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long cycles_t;
 
diff -puN include/asm-ppc64/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-ppc64/timex.h
--- 25/include/asm-ppc64/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.290379080 -0700
+++ 25-akpm/include/asm-ppc64/timex.h	2004-09-20 11:30:28.304376952 -0700
@@ -12,10 +12,6 @@
 #define _ASMPPC64_TIMEX_H
 
 #define CLOCK_TICK_RATE	1193180 /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long cycles_t;
 
diff -puN include/asm-ppc/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-ppc/timex.h
--- 25/include/asm-ppc/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.291378928 -0700
+++ 25-akpm/include/asm-ppc/timex.h	2004-09-20 11:30:28.304376952 -0700
@@ -11,10 +11,6 @@
 #include <asm/cputable.h>
 
 #define CLOCK_TICK_RATE	1193180 /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long cycles_t;
 
diff -puN include/asm-s390/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-s390/timex.h
--- 25/include/asm-s390/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.293378624 -0700
+++ 25-akpm/include/asm-s390/timex.h	2004-09-20 11:30:28.305376800 -0700
@@ -12,10 +12,6 @@
 #define _ASM_S390_TIMEX_H
 
 #define CLOCK_TICK_RATE	1193180 /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long long cycles_t;
 
diff -puN include/asm-sh/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-sh/timex.h
--- 25/include/asm-sh/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.294378472 -0700
+++ 25-akpm/include/asm-sh/timex.h	2004-09-20 11:30:28.305376800 -0700
@@ -7,10 +7,6 @@
 #define __ASM_SH_TIMEX_H
 
 #define CLOCK_TICK_RATE		(CONFIG_SH_PCLK_FREQ / 4) /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long long cycles_t;
 
diff -puN include/asm-sparc64/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-sparc64/timex.h
--- 25/include/asm-sparc64/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.295378320 -0700
+++ 25-akpm/include/asm-sparc64/timex.h	2004-09-20 11:30:28.306376648 -0700
@@ -9,10 +9,6 @@
 #include <asm/timer.h>
 
 #define CLOCK_TICK_RATE	1193180 /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 /* Getting on the cycle counter on sparc64. */
 typedef unsigned long cycles_t;
diff -puN include/asm-sparc/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-sparc/timex.h
--- 25/include/asm-sparc/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.297378016 -0700
+++ 25-akpm/include/asm-sparc/timex.h	2004-09-20 11:30:28.305376800 -0700
@@ -7,10 +7,6 @@
 #define _ASMsparc_TIMEX_H
 
 #define CLOCK_TICK_RATE	1193180 /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 /* XXX Maybe do something better at some point... -DaveM */
 typedef unsigned long cycles_t;
diff -puN include/asm-v850/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-v850/timex.h
--- 25/include/asm-v850/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.298377864 -0700
+++ 25-akpm/include/asm-v850/timex.h	2004-09-20 11:30:28.306376648 -0700
@@ -7,10 +7,6 @@
 #define __V850_TIMEX_H__
 
 #define CLOCK_TICK_RATE	1193180 /* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((long)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long cycles_t;
 
diff -puN include/asm-x86_64/timex.h~cleanup-remove-unused-definitions-from-timexh include/asm-x86_64/timex.h
--- 25/include/asm-x86_64/timex.h~cleanup-remove-unused-definitions-from-timexh	2004-09-20 11:30:28.299377712 -0700
+++ 25-akpm/include/asm-x86_64/timex.h	2004-09-20 11:30:28.306376648 -0700
@@ -13,10 +13,6 @@
 #include <asm/hpet.h>
 
 #define CLOCK_TICK_RATE	PIT_TICK_RATE	/* Underlying HZ */
-#define CLOCK_TICK_FACTOR	20	/* Factor of both 1000000 and CLOCK_TICK_RATE */
-#define FINETUNE ((((((int)LATCH * HZ - CLOCK_TICK_RATE) << SHIFT_HZ) * \
-	(1000000/CLOCK_TICK_FACTOR) / (CLOCK_TICK_RATE/CLOCK_TICK_FACTOR)) \
-		<< (SHIFT_SCALE-SHIFT_HZ)) / HZ)
 
 typedef unsigned long long cycles_t;
 
_
